src/api/EscargotPublic.h

```diff
@@ -1039,6 +1039,8 @@ class ESCARGOT_EXPORT ObjectRef : public PointerValueRef {
                                             ExposableObjectGetOwnPropertyCallback getOwnPropertyCallback, ExposableObjectDefineOwnPropertyCallback defineOwnPropertyCallback,
                                             ExposableObjectEnumerationCallback enumerationCallback, ExposableObjectDeleteOwnPropertyCallback deleteOwnPropertyCallback);
 
+    static bool setIntegrityLevel(ExecutionStateRef* state, ObjectRef* obj, bool isSealed);
```

Comment:
IMO we can make this function as just member function not static

Reply:
Good point! Fixed it.